src/main/java/twilightforest/config/TFClientConfig.java

```diff
@@ -62,6 +63,11 @@ public TFClientConfig(ModConfigSpec.Builder builder) {
 			.comment("Renders how many fortification shields are currently active on your player above your armor bar. Turn this off if you find it intrusive or other mods render over/under it.")
 			.define("fortificationShieldIndicator", true);
 
+		showFortificationShieldIndicatorInCreative = builder
+			.translation(TFConfig.CONFIG_ID + "shield_indicator_creative")
+			.comment("Enables shield indicator in creative for debugging.")
```

Comment:
While it's implied by the config name and the config above, should probably specify it as the fortification shield indicator instead of just shield indicator.